docs/api_definitions.md

```diff
@@ -42,12 +42,8 @@ To check the supported features of the API Definitions CRD version you're curren
 | Keyless (Open)                | ✅        | v0.1           | -        | [Sample](./../config/samples/httpbin.yaml) |
 | Static Bearer Token           | ✅        | v0.1           | -        | [Sample](./../config/samples/httpbin_protected.yaml) |
 | JWT                           | ✅️        | v0.5           | -        | [Sample](./../config/samples/jwt-auth) |
-| OAuth2 - Client Credentials   | ✅️        | v0.6           | -        | [Sample](./../config/samples/oauth2/client_credentials.yaml) |
```

Comment:
@caroltyk since this one has already an example, maybe we can add back this line and rephrase to explain it's an internal oauth server?

Reply:
@letzya Can you suggest the changes?